FirebaseAuth/Tests/Sample/SwiftApiTests/FacebookTests.swift

```diff
@@ -82,6 +106,35 @@ class FacebookTests: TestsBase {
     self.deleteFacebookTestingAccountbyID(facebookAccountID)
   }
 
+#if compiler(>=5.5) && canImport(_Concurrency)
+  @available(iOS 15, tvOS 15, macOS 12, watchOS 8, *)
+  func testLinkAnonymousAccountToFacebookAccountAsync() async throws {
+    let auth = Auth.auth()
+    try await self.signInAnonymouslyAsync()
+    let userInfoDict = try await self.createFacebookTestingAccountAsync()
+    guard let facebookAccessToken = userInfoDict["access_token"] as! String? else {
+      XCTFail("Failed to get facebookAccessToken")
+      return
+    }
+    guard let facebookAccountID = userInfoDict["id"] as! String? else {
+      XCTFail("Failed to get facebookAccountID")
+      return
+    }
```

Comment:
Not quite. but the following works:

    let facebookAccessToken: String = try XCTUnwrap(userInfoDict["access_token"]) as! String
    let facebookAccountID: String = try XCTUnwrap(userInfoDict["id"]) as! String

Reply:
Neat, thanks! I think it's fine as is, but for my own curiousity for the future I'm wondering if we can put the cast inside the `XCTUnwrap` to make the test fail instead of crash:

```
let facebookAccessToken: String = try XCTUnwrap(userInfoDict["access_token"]  as? String)
```